src/main/java/org/tikv/common/TiSession.java

```diff
@@ -159,23 +159,46 @@ private static VersionInfo getVersionInfo() {
 
   private synchronized void warmUp() {
     long warmUpStartTime = System.currentTimeMillis();
+    warmUpStore();
+    warmUpRegion();
+    warmUpGrpc();
+    logger.info(
+        String.format("warm up duration %d ms", System.currentTimeMillis() - warmUpStartTime));
```

Comment:
should we use `System.nanoTime()` to avoid issues #404 happen again?

Reply:
we do not need an accurate measure here.

Reply:
since you are logging the time, and System.currentTimeMillis() can be misleading, I think we should avoid using it to record time durations.